services/app-web/src/pages/StateSubmission/ReviewSubmit/ReviewSubmit.tsx

```diff
@@ -156,14 +204,58 @@ export const ReviewSubmit = ({
             <Modal
                 modalRef={modalRef}
                 id="review-and-submit"
-                modalHeading="Ready to submit?"
+                modalHeading={unlocked ? 'Summarize changes' : 'Ready to submit?'}
                 submitButtonProps={{ className: styles.submitButton }}
-                onSubmit={handleFormSubmit}
+                onSubmitText={unlocked ? 'Resubmit' : undefined}
+                onSubmit={() => {
+                    setFocusErrorsInModal(true)
+                    formik.handleSubmit()
+                }}
             >
-                <p>
-                    Submitting this package will send it to CMS to begin their
-                    review.
-                </p>
+                <form>
```

Comment:
The form element should be inside the `unlock ? (this JSX): (that JSX)` logic. We don't have a form element on the page at all if we aren't looking at an unlocked submission that will be resubmitted. That will lead to funky behaviors in assistive devices.